### DISCUSSION TITLE
Alerts are not showing up in SO

### DISCUSSION THREAD

**Discussion by molfar101** [Security-Onion-Solutions/securityonion#8226] [Unsupported Versions]
Hello,

I have issue with my SO - the alerts are not showing up in the Alert tab. I use playbook plays for triggering alerts - two plays are active for testing and should trigger lots of alerts. The logs are coming in from the event collector - I can see all events in the Hunt tab. I copied  the query in the ElastAlert rule file to Hunt and could see all events that match the rule but nothing shows up in the Alert tab. After checking the logs (opt/so/log/elastalert) I determined that there are hits and the alerts are sent (x matches, x alerts sent). I rebooted elastalert, playbook, and the entire system - nothing helped. I have a distributed architecture running on Ubuntu 18.4. SO version is 2.3.30. Please advise what might be causing this issue.

**Comment by dougburks** [comment]
> SO version is 2.3.30

If you're still running version 2.3.30, then the first thing I'd do is update to the latest version.

**Reply by molfar101** [reply]
I've been thinking about it for awhile but I don't have a testing environment to make sure nothing breaks. Therefore, I want to postpone the update until I build a testing environment. If nothing helps, I will update it but this post is my last hope to fix it.
